docs/user_manual/source/rvfi.rst

```diff
@@ -72,29 +72,58 @@ The trap signal indicates that a synchronous trap has ocurred and side-effects c
+  |                              |           |     |     |     |       |        +---------+-----------------+-----------------------------------------------------------------------+
+  |                              |           |     |     |     |       |        | 0x2     | mcause, mepc    | Load attempt with address failing PMP check.                          |
+  +------------------------------+-----------+-----+-----+-----+-------+--------+---------+-----------------+-----------------------------------------------------------------------+
+  | Store/AMO Access Fault       | Exception | 1   | 1   | X   | 0x7   | X      | 0x0     | mcause, mepc    | Non-naturally aligned store access attempt to an I/O region           |
+  |                              |           |     |     |     |       |        +---------+-----------------+-----------------------------------------------------------------------+
+  |                              |           |     |     |     |       |        | 0x1     | mcause, mepc    | SC or AMO attempt to region without atomic support.                   |
+  |                              |           |     |     |     |       |        +---------+-----------------+-----------------------------------------------------------------------+
+  |                              |           |     |     |     |       |        | 0x2     | mcause, mepc    | Store attempt with address failing PMP check.                         |
+  +------------------------------+-----------+-----+-----+-----+-------+--------+---------+-----------------+-----------------------------------------------------------------------+
+  | Environment Call             | Exception | 1   | 1   | X   | 0x8   | X      | 0x0     | mcause, mepc    | ECALL executed from User mode                                         |
+  |                              |           |     |     |     +-------+--------+---------+-----------------+-----------------------------------------------------------------------+
+  |                              |           |     |     |     | 0xB   | X      | 0x0     | mcause, mepc    | ECALL executed from Machine mode                                      |
+  +------------------------------+-----------+-----+-----+-----+-------+--------+---------+-----------------+-----------------------------------------------------------------------+
+  | Instruction Bus Fault        | Exception | 1   | 1   | X   | 0x30  | X      | 0x0     | mcause, mepc    | OBI bus error on instruction fetch                                    |
+  +------------------------------+-----------+-----+-----+-----+-------+--------+---------+-----------------+-----------------------------------------------------------------------+
+  | Breakpoint to debug          | Debug     | 1   | X   | 1   | 0x0   | 0x1    | 0x0     | dpc, dcsr       | EBREAK from non-debug mode executed with  dcsr.ebreakm == 1           |
```

Comment:
[8:3] should be X'es for all debug causes I think?

Reply:
Yes, typo. Will fix.